packages/network/samba/config/smb.conf

```diff
@@ -27,18 +27,13 @@
   security = share
   guest account = root
   socket options = TCP_NODELAY IPTOS_LOWDELAY
-  smb ports = 445
   max protocol = SMB2
   min receivefile size = 16384
   deadtime = 30
-  os level = 20
   mangled names = no
   syslog only = yes
   syslog = 2
-  name resolve order = lmhosts wins bcast host
-  preferred master = auto
```

Comment:
I'd say you really shouldn't be changing the preferred master and os level unless you know what you're doing, and if you do then you probably don't need them listed. Anyways, I'll put them back if you like.